src/core.js

```diff
@@ -2135,7 +2144,50 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
    * @param {Number} column Visual column index.
    * @returns {Number} Returns physical column index.
    */
-  this.toPhysicalColumn = column => this.columnIndexMapper.getPhysicalIndex(column);
+  this.toPhysicalColumn = column => this.columnIndexMapper.getPhysicalFromVisualIndex(column);
+
+  /**
+   * // Changed by @wszymanski as there was link to the not existing anymore RecordTranslator.
+   * @TODO Description
+   */
+  this.renderedToPhysicalColumn = (column) => {
```

Comment:
Since we're using `rendered` indexes only internally, is it necessary to add those methods to public APIs? I've seen that some plugins use them, but they have access to `indexMapper` anyway.